src/main/resources/application.properties

```diff
@@ -17,7 +17,7 @@ server.error.whitelabel.enabled=false
 # Jackson
 spring.jackson.default-property-inclusion=non_null
 spring.jackson.deserialization.fail-on-null-creator-properties=true
-spring.jackson.date-format=dd-MM-yyyy
+spring.jackson.date-format=dd-MM-yyyy HH:mm:ss
```

Comment:
Kind of nitty gritty here, but because the plan here is to use these for event time, maybe we could use hours and minutes only?